use-shopping-cart/package.json

```diff
@@ -1,16 +1,34 @@
 {
   "name": "use-shopping-cart",
-  "version": "2.4.2",
+  "version": "3.0.0-beta.2",
   "description": "Shopping cart state and logic for Stripe",
   "author": "dayhaysoos",
   "license": "MIT",
   "repository": "dayhaysoos/use-shopping-cart",
-  "main": "dist/index.js",
-  "types": "./src/index.d.ts",
-  "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
-  "unpkg": "dist/index.umd.js",
-  "umd": "dist/index.umd.js",
+  "main": "dist/react.js",
+  "types": "./src/react.d.ts",
+  "module": "dist/react.es.js",
+  "jsnext:main": "dist/react.es.js",
+  "unpkg": "dist/react.umd.js",
+  "umd": "dist/react.umd.js",
+  "exports": {
+    ".": {
+      "require": "./dist/react.js",
+      "node": "./dist/react.js",
+      "import": "./dist/react.es.js",
+      "browser": "./dist/react.umd.js"
+    },
+    "./core": {
+      "require": "./dist/core.js",
+      "node": "./dist/core.js",
+      "import": "./dist/core.es.js",
+      "browser": "./dist/core.umd.js"
+    },
+    "./utilities": {
+      "require": "./utilities/serverless.js",
+      "node": "./utilities/serverless.js"
+    }
```

Comment:
Wouldn't `"./serverless"` make more sense here since this only adds an exports field for the `serverless.js` file and not any of the other utilities?

Reply:
I think line 27 used to be `./serverless` but it didn't work because there was no `serverless` directory so I was getting errors around it not existing. I believe the other utilities are available the same way, I gotta double check though

Reply:
Hm @dayhaysoos, if someone is using:

```js
import /* ... */ from 'use-shopping-cart/serverless'
```

It should work if they're using any modern bundler. If they aren't they should be doing exact filepath imports like:

```js
import /* ... */ from 'use-shopping-cart/utilities/serverless.js'
```

This is also a feature of Node.js. It recognizes the exports field in newer versions. If someone is using an outdated version, they should use the full path.

That being said, changing it to utilities shouldn't have fixed anything since there's no `utilities/index.js` file.

Reply:
I'm gonna have to take a look at it again, I was really struggling to get this working as it was before and this was the only thing that did it. Gonna poke around and see what I come up with